plugins/analysis-icu/src/main/java/org/elasticsearch/index/analysis/IcuNormalizerTokenFilterFactory.java

```diff
@@ -38,8 +38,10 @@
  * <p>The {@code unicodeSetFilter} attribute can be used to provide the UniCodeSet for filtering.</p>
  */
 public class IcuNormalizerTokenFilterFactory extends AbstractTokenFilterFactory implements MultiTermAwareComponent {
-    private static final DeprecationLogger deprecationLogger =
+
+    private static final DeprecationLogger DEPRECATION_LOGGER =
```

Comment:
We've been using `deprecationLogger` because the logger does things rather than is a constant.

Reply:
Have changed it back